sycl/test/basic_tests/image.cpp

```diff
@@ -0,0 +1,67 @@
+// RUN: %clang -fsycl -std=c++11 %s -o %t.out -lstdc++ -lOpenCL -lsycl
+// RUNx: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+//==------------------- image.cpp - SYCL image basic test -----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+#include <iostream>
+#include <vector>
+
+#include "../helpers.hpp"
```

Comment:
I guess it is bad practice use `..` in includes.

Reply:
Could you suggest better solution? I don't think adding directory with helper to include patch in cmake files is good solution as in this case we can't just compile test in source directory like this:
clang++ -fsycl ./sycl/test/basic_test/image.cpp

Reply:
do we plan to use it anythere excep image.cpp? May be it is reasonable to move it to basic_tests directory and remove .. from path?

Reply:
Yes, I would like to use common structures in all test instead of implementing them in each test.